### DISCUSSION TITLE
Log Maps

### DISCUSSION THREAD

**Discussion by r3chn3r** [magicbug/Cloudlog#1953] [General]
Is it possible for Cloudlog to requisition gridsquare from qrz or somewhere other than manual entry? 

Every new contact is in Wisconsin if I don't manually add a location for it.

**Comment by r3chn3r** [comment]
nevermind! lol....

**Comment by magicbug** [comment]
Figured it out?

**Comment by r3chn3r** [comment]
Yup. Thank you.

**Comment by elitegoodguy** [comment]
What was the solution?  I've noticed this too and I guess I haven't figured this out?

**Reply by elitegoodguy** [reply]
Maybe I did figure it out huh...  I just checked a QSO that I recently logged and it's showing up the correct map location.  But my older ones are showing up in WI if there's no Gridsquare setup
